pkgs/development/libraries/vtk/generic.nix

```diff
@@ -89,6 +89,12 @@ in stdenv.mkDerivation rec {
     sed -i 's/fprintf(output, shift)/fprintf(output, "%s", shift)/g' ./ThirdParty/libxml2/vtklibxml2/xpath.c
   '';
 
+  postInstall = optionalString enablePython ''
+    substitute ${./vtk.egg-info} \
+      $out/${python.sitePackages}/vtk-${version}.egg-info \
```

Comment:
`pythonInterpreter` should be replaced by `python` in the whole file. Also in `python-packages.nix` please replace `pythonInterpreter = python;` with `inherit (self) python;`.